composer.json

```diff
@@ -15,25 +15,34 @@
     "php": ">=7.4",
     "symfony/process": "~2.5|~3.0|~4.4|^6",
     "symfony/dependency-injection": "~2.6|~3.0|~4.4|^6",
-    "drupal/core-utility": "^8.4 || ^9 || ^10.0.0-alpha1"
+    "drupal/core-utility": "^8.4 || ^9 || ^10@beta"
   },
   "require-dev": {
+    "composer/installers": "^2.1",
     "drupal/coder": "~8.3.0",
     "phpspec/phpspec": "~2.0 || ~4.0 || ~6.1 || dev-main",
     "phpunit/phpunit": "~6.0 || ~7.0 || ^9",
-    "mockery/mockery": "^0.9.4",
+    "mockery/mockery": "^1.5",
+    "drupal/core-composer-scaffold": "^10@beta",
+    "drupal/core-recommended": "^10@beta",
+    "drupal/mailsystem": "^4.4",
     "drush-ops/behat-drush-endpoint": "*",
     "php-parallel-lint/php-parallel-lint": "^1.0",
-    "dms/phpunit-arraysubset-asserts": "^0.4.0"
+    "dms/phpunit-arraysubset-asserts": "^0.4.0",
+    "mglaman/drupal-check": "^1",
+    "palantirnet/drupal-rector": "^0.13"
   },
   "scripts": {
     "test": [
       "composer validate --no-interaction",
       "parallel-lint src spec tests",
       "phpunit",
       "phpspec run -f pretty --no-interaction",
-      "phpcs --standard=./phpcs-ruleset.xml ."
-
+      "phpcs --standard=./phpcs-ruleset.xml --ignore=drupal .",
```

Comment:
Let's not add the `--ignore` option inline. Let's instead update the ignored paths in `phpcs-ruleset.xml`:

```
<exclude-pattern>./drupal/*</exclude-pattern>
```